SPECS/libnotify/disable-gtk.patch

```diff
@@ -26,24 +26,3 @@ index 088b7ed..bafeb7d 100644
  
  conf = configuration_data()
  conf.set_quoted('VERSION', meson.project_version())
```

Comment:
let's remove this patch. Since meson command in Spec file can take care of this.

Reply:
I read it wrong. You've already removed it 👍